README.md

```diff
@@ -1 +1,83 @@
-# Event-Finder
+# Event Finder
+
+> Tge Event Finder application is an appliation where you can find A list of events with details about each event. 
+- Display a list of Event.
+- Filter Events By Category (Music, Sports).
+- See Details about a specific Event.
+
```

Comment:
- [ ] kindly ensure that A link to the deployed project is present in the README. this will ease interested party reach see your work faster